<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WarehouseSystemAddCustomerDialog</class>
 <widget class="QDialog" name="WarehouseSystemAddCustomerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление покупателя | ООО &quot;Нинель&quot;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="4" column="0">
       <widget class="QPushButton" name="pbSave">
        <property name="text">
         <string>Сохранить</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="leINN_Header">
          <property name="text">
           <string>ИНН</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leINN">
          <property name="inputMask">
           <string/>
          </property>
          <property name="maxLength">
           <number>999999999</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lCheckINN">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="leAddressHeader">
          <property name="text">
           <string>Адрес</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leAddress"/>
        </item>
        <item>
         <widget class="QLabel" name="lCheckAddress">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="lName">
          <property name="text">
           <string>Наименование</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leName"/>
        </item>
        <item>
         <widget class="QLabel" name="lCheckName">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="lePhoneHeader">
          <property name="text">
           <string>Телефон</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lePhone">
          <property name="maxLength">
           <number>99</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lCheckPhone">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="pbClose">
        <property name="text">
         <string>Закрыть</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leName</tabstop>
  <tabstop>leINN</tabstop>
  <tabstop>lePhone</tabstop>
  <tabstop>leAddress</tabstop>
  <tabstop>pbSave</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
